node/resolvers/checkout/attachmentsHelper.ts

```diff
@@ -43,38 +47,71 @@ interface AddOptionsLogicInput {
   oldItems: OrderFormItem[]
 }
 
-const addAssemblyBody = (items: OptionRequestUnit[]) => ({
-  composition: {
-    items: map<OptionRequestUnit>(omit(['options']) as any, items),
-  },
-  noSplitItem: true,
-})
+interface AssemblyOptionBody {
+  noSplitItem?: boolean
```

Comment:
what does `noSplitItem` mean? and is it an internal term or does it come from the API?

Reply:
That's Checkout's API